subprojects/core/src/main/java/org/gradle/internal/service/scopes/BuildScopeServices.java

```diff
@@ -176,7 +175,6 @@
 public class BuildScopeServices extends DefaultServiceRegistry {
     public BuildScopeServices(final ServiceRegistry parent) {
         super(parent);
-        addProvider(new BuildCacheServices());
```

Comment:
Does that mean we registered `BuildCacheServices` twice?

Reply:
I think it only meant that we were looking for `create*()` etc. methods, and found none.